File: server/groute/src/main/java/com/ssafy/groute/dto/board/Comment.java
package com.ssafy.groute.dto.board;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Setter
@Getter
@NoArgsConstructor
@AllArgsConstructor
public class Comment {
    private int id;
    private String content;
    private int level;
    private int groupNum;
    private int order;
    private int boardDetailId;
    private String userId;
}

File: client/Groute/app/src/main/java/com/ssafy/groute/src/dto/Comment.kt
package com.ssafy.groute.src.dto

import java.io.Serializable


data class Comment(
    val id: Int,
    val boardDetailId: Int,
    var content: String,
    val groupNum: Int,
    val level: Int,
    val order: Int,
    val userId: String
) : Serializable
{
    constructor(boardDetailId: Int, content: String, groupNum: Int, userId: String): this(0, boardDetailId, content, groupNum, 0, 0, userId)
    constructor(boardDetailId: Int, content: String, groupNum: Int, userId: String, level: Int, order: Int): this(0, boardDetailId, content, groupNum, level, order, userId)
}
